app/models/form/sales/questions/staircase_bought.rb

```diff
@@ -8,6 +8,6 @@ def initialize(id, hsh, page)
     @width = 5
     @min = 0
     @max = 100
-    @suffix = " percent"
+    @suffix = "%"
```

Comment:
@kosiakkatrina I changed this for consistency with other questions on the assumption that symbols are more concise and also a bit more consistent with eg £ prefix. Let me know if it should definitely be "percent"

Reply:
% looks okay, you might need to rebase and fix some tests in `validate_numeric_min_max` tests